node/src/components/consensus/highway_core/state.rs

```diff
@@ -568,13 +568,11 @@ impl<C: Context> State<C> {
         self.pings[creator] = self.pings[creator].max(timestamp);
     }
 
-    /// Returns `true` if the latest timestamp we have is less than one maximum round length older
-    /// than the given timestamp.
-    ///
-    /// This is to prevent ping spam: If the incoming ping is only slightly newer than a unit or
-    /// ping we have already received, we drop it without forwarding it to our peers.
+    /// Returns `true` if the latest timestamp we have is older than the given timestamp.
```

Comment:
I'm trying to understand.

If the ping was not created by us, it seems the old behavior makes sense.

And if the ping was from us, perhaps we could prevent ourselves from forwarding it elsewhere?

Reply:
Looks like we aren't forwarding, so this is the correct behavior.